src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -129,6 +129,11 @@ object Parameters {
   val PARAM_USE_EXPONENTIAL_BACKOFF: String = knownParam(
     "use_exponential_backoff"
   )
+  // Internal option to remove (") in stage table name.
+  // It may be removed without any notice in any time.
+  val PARAM_INTERNAL_REMOVE_QUOTE_FOR_STAGE_TABLE_NAME: String = knownParam(
```

Comment:
rename to PARAM_INTERNAL_STAGING_TABLE_NAME_REMOVE_QUOTES_ONLY

Reply:
thanks. Update it.